third_party/move/move-compiler-v2/tests/live-var/v1-commands/branch_assigns_then_moves.exp

```diff
@@ -0,0 +1,111 @@
+============ initial bytecode ================
```

Comment:
Can you leave all this live-var tests out? This is refactored in the not yet reviewed (sigh) PRs and the tests have to go somewhere else. if we land yours there is a risk we forget.

Reply:
done